### PR TITLE
PHP 7.4 and 8.0 support

### PR DESCRIPTION
**Pull request by GrahamCampbell**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #160
| License         | MIT

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Welp, this technically works :trollface:. Avoids the problem of the skipping package not being maintained.

File: spec/autoload.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
     ]);
 }
 
-if (PHP_VERSION_ID >= 70300 && PHP_VERSION_ID < 70400) {
+if (PHP_VERSION_ID >= 70300) {
```
Comment (dbu) [inline spec/autoload.php:11]: this was a bit odd. @Nyholm would you remember if this was done for a specific reason?

if i understand correctly, this is only for the tests, so we could rename the classes without isuses. if the semantic of the if is correct, AtLeastPHP73 and LessThanPHP73 would be better names.

if we would ever need an upper bound, we can introduce it and rename the classes again.

Comment (GrahamCampbell) [reply spec/autoload.php:11]: At the time Puli didn;t work with PHP 7.3 yet, and that was the latest version. 2 years later, we are now in the siutation where 7.4 and 8.0 don't work either. I guess I can rename this class if we want. Let me know what name you prefer. :)

Comment (GrahamCampbell) [reply spec/autoload.php:11]: As for what this class does, yes, it exists so we can skip the puli tests.

**Comment by GrahamCampbell** [comment]
@dbu This is ready now. :)

File: spec/autoload.php (hunk 1 above)
Comment (dbu) [reply spec/autoload.php]: yep. lets call them AtLeastPHP73 and LessThanPHP73? or possibly: PuliCompatible and PuliIncompatible (if we even need both and not only one)

Comment (GrahamCampbell) [reply spec/autoload.php]: Already renamed it around an hour ago btw. :)

**Pull request merged**